tests/integration/app-charm/charmcraft.yaml

```diff
@@ -2,15 +2,30 @@
 # See LICENSE file for licensing details.
 
 type: charm
-
-parts:
-  charm:
-    charm-binary-python-packages:
-      - psycopg2-binary
-      - mysql-connector-python ~= 8.0.33
-
 bases:
   - name: ubuntu
     channel: "22.04"
     architectures: [amd64]
 
+parts:
+  charm:
+    build-packages:
+      - libpq-dev
+    override-build: |
+      # Convert subset of poetry.lock to requirements.txt
+      curl -sSL https://install.python-poetry.org | python3 -
+      /root/.local/bin/poetry export --only main,charm-libs --output requirements.txt
+
+      craftctl default
+    charm-binary-python-packages:
+      - mysql-connector-python==9.0.0  # renovate: pypi
```

Comment:
We either need to figure out how to build or keep the version the same as in poetry.

Reply:
I would stay here with 8.0.33 (like we do in MySQL charms).
9.0.0 will work as well, but we are on MySQL 8.0 currently.

Reply:
If so should we disable major and minor updates in renovate for the connector?

Reply:
It is a test aplication to test data-integrator.
IMHO, we can go here with the latest m-c-p and see how it behaves (as no risks here). K.I.S.S. ;-)